### PR TITLE
Benchmarks: about the data Page

### PR DESCRIPTION
**Pull request by g-m-taylor**
#### Tests
Does this PR include tests for the changes introduced?
- [x] Yes
- [ ] No, I found it difficult to test
- [ ] No, they are not required for this change

### CODE REVIEW

File: src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.html [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    <div class="govuk-inset-text">
+      Note, to view the benchmark for turnover, the number of staff records added must be the same as the total number of staff you have.
+    </div>
+    <h2 class="govuk-heading-m"> Comparison group</h2>
+    <p>Your comparison group data:  </p>
+      <ul>
+        <li class="govuk-list govuk-list--bullet"><p>is updated every month</p></li>
+        <li class="govuk-list govuk-list--bullet"><p>has passed extensive quality checks that cross-validate and remove outliers</p></li>
+        <li class="govuk-list govuk-list--bullet" *ngIf="meta && meta.staff && meta.workplaces"><p>is {{formatNumber(meta.staff)}} staff from {{formatNumber(meta.workplaces)}} {{pluralizeWorkplaces(meta.workplaces)}} providing the same main service as you in your local authority.</p></li>
+      </ul>
```
Comment (aaron-russell) [inline src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.html:19]: You can just add those classes to the UL. The LI's will inherit it

File: src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    protected benchmarksService: BenchmarksService,
+    protected backService: BackService
+  ) {}
+
+  ngOnInit() {
+    this.subscriptions.add(
+      this.benchmarksService.getMeta(this.route.snapshot.params.establishmentID).subscribe(
+        (data) => {
+          if (data) {
+            console.log(data);
```
Comment (aaron-russell) [inline src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts:28]: Console log

File: src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+      ))
+    this.backService.setBackLink({
+      url: ['dashboard'],
+      fragment: 'benchmarks',
+    });
+  }
+
+  public formatNumber(data) {
+    return  data.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  }
```
Comment (aaron-russell) [inline src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts:41]: I think you can juse pipe it to a global formatter?

**Review by aaron-russell — CHANGES_REQUESTED** [review]

File: src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts (hunk 3 above)
Comment (chadtomkiss-madetech) [reply src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts:41]: @aaron-russell yeah, you can do:

```
{{ meta.staff | number }}
```

in the template

File: src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      fragment: 'benchmarks',
+    });
+  }
+
+  public pluralizeWorkplaces(workplaces){
+    return workplaces > 1 ? 'workplaces' : 'workplace'
+  }
+
+  ngOnDestroy() {
+  }
```
Comment (aaron-russell) [inline src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts:43]: You're not doing anything onDestroy, is it needed?

Comment (aaron-russell) [reply src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts:43]: Or are you meaning to unsubscribe on destroy?

**Review by aaron-russell — CHANGES_REQUESTED** [review]

File: src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts (hunk 4 above)
Comment (g-m-taylor) [reply src/app/shared/components/benchmarks-tab/about-the-data/about-the-data.component.ts:43]: I will need to for the unsubscribe, but also TS complains when it doesnt exist

**Pull request merged**